auth0-express-js-sample/client/src/components/Search/AutoComplete.js

```diff
@@ -24,15 +32,6 @@ const AutoComplete = ({ data, onSelect }) => {
         });
     };
 
-    useEffect(() => {
-        if (cursor < 0 || cursor > suggestions.length || !searchResultRef) {
-            return () => {};
-        }
-
-        let listItems = Array.from(searchResultRef.current.children);
-        listItems[cursor] && scrollIntoView(listItems[cursor].offsetTop);
-    }, [cursor]);
-
     const suggestions = useMemo(() => {
```

Comment:
line 35 defines `suggestions`, which is used in line 28 before it was defined - causing warnings.
I simply moved it downwards.